<?xml version="1.0"?>
<launch>
    <node pkg = "navigator_bringup"   type = "keypad.py"        name = "keypad"/>
    <node pkg = "navigator_bringup"   type = "gps_tf.py"        name = "gps_tf"          output = "screen"/>
    <node pkg = "rosserial_python"    type = "serial_node.py"   name = "serial_nano"     output = "screen">
        <param name = "baud"  value = "115200"/>
        <param name = "port"  value = "/dev/ttyUSB0"/>
    </node>
    <node pkg = "rosserial_python"    type = "serial_node.py"   name = "serial_uno"      output = "screen">
        <param name = "baud"  value = "57600"/>
        <param name = "port"  value = "/dev/ttyACM0"/>
    </node>
    <include file = "$(find navigator_bringup)/launch/3dsensor.launch">
        <arg name = "rgb_processing"     value = "false" />
        <arg name = "depth_registration" value = "false" />
        <arg name = "depth_processing"   value = "false" />
        <arg name = "scan_topic"         value = "/scan" />
  </include>
</launch>